deploy/raft-storage-controller.yaml

```diff
@@ -181,7 +168,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: raft-storage-controller
-    namespace: kube-system
+    namespace: default
```

Comment:
If this is a cluster scoped controller it should be deployed in the `kube-system` namespace as is convention for all controllers